### DISCUSSION TITLE
How can I contribute?

### DISCUSSION THREAD

**Discussion by Nicola-Bini** [python-telegram-bot/python-telegram-bot#2774] [Q&A]
Hello everyone, 

I love the package and I would like to contribute to it. I have decent experience in Python but I am not an expert. Is there some beginner friendly bugs I can help with?

Thank you!

**Comment by Poolitzer** [comment]
Hi. Right before October, we label beginner friendly issues with [hacktoberfest](https://github.com/python-telegram-bot/python-telegram-bot/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest). These are always worth checking out, but only #2524 is open and python related...

Otherwise, we are in the process of switching to asyncio, and right before the async change itself, so not too many low hanging fruits open right now Im afraid.

**Reply by Bibo-Joshi** [reply]
#2633 has several components though & some even need a bit of coding :)
